CREDITS

```diff
@@ -1,305 +1,307 @@
-F: cph
+This is a list of all the contributors that have submitted their work to the Freedoom project, since the first release in 2003, during the development phase from 2001 and onwards. It's a long list.
 N: Colin Phipps
 S: cph
 E: cph@cph.demon.co.uk
 W: http://www.cph.demon.co.uk/
 D: Binary lumps (playpal, colormap etc)
 
-F: julian
+
```

Comment:
Please remove duplicate empty lines; it's sufficient to just have a single empty line between each contributor.